integrations/tor_analyzers/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM python:3.8-slim
+
+ENV PROJECT_PATH /opt/deploy
+ENV LOG_PATH /var/log/intel_owl/tor_analyzers
+ENV USER tor-user
+
+# Add a new low-privileged user
+RUN useradd -r -s /sbin/nologin ${USER}
+
+# update and install packages
+RUN DEBIAN_FRONTEND=noninteractive apt-get update -qq \
+    && apt-get install --no-install-recommends -y git build-essential gcc pandoc curl tor \
```

Comment:
I see we are installing `tor` here which makes sense but where/when is the tor daemon being launched ?

Reply:
We launch the tor daemon from the entry point script, before the python api starts.

Reply:
Oh! I missed that file. Thanks for the clarification.